lib/features/chat/services/chat_gpt_service.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:chat_gpt_sdk/chat_gpt_sdk.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter_chat_types/flutter_chat_types.dart' as types;
+import 'package:flutter_firebase_chat_core/flutter_firebase_chat_core.dart';
+
+import '../../../resources/constants/chat_gpt_user.dart';
+
+class ChatGptService {
+  static final openAI = OpenAI.instance.build(
+    token: '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',
```

Comment:
Replace this secret key to use this:

![Screenshot 2023-02-24 at 07 58 10](https://user-images.githubusercontent.com/39690358/221065843-1a8ea139-ea45-4516-8024-0f2443622b4f.png)